src/start_server.ts

```diff
@@ -143,6 +152,23 @@ export function getApp(options: ServerOptions): express.Express {
 
   app.use('/components/', polyserve);
 
+  if (options.proxyRoot && options.proxyTarget) {
+    const apiProxy = httpProxy.createProxyServer();
+    const proxyHandler = (req: express.Request, res: express.Response) => {
+      // Remove proxied part from the url
+      req.url = req.url.slice(options.proxyRoot.length + 1);
+      apiProxy.web(req, res, { target: options.proxyTarget }, (error: proxy.ProxyError) => {
+        res.statusCode = error.status || 500;
+        res.end(error.message);
+      });
+    };
+
+    app.post(`/${options.proxyRoot}/*`, proxyHandler);
```

Comment:
We should either validate that the path doesn't include '*', '+' and '?', or escape them before handing to express.